source/val/validate_id.cpp

```diff
@@ -2086,7 +2086,7 @@ spv_result_t CheckIdDefinitionDominateUse(const ValidationState_t& _) {
             if (use->opcode() == SpvOpPhi) {
               phi_instructions.insert(use);
             } else if (!block->dominates(*use->block())) {
-              return _.diag(SPV_ERROR_INVALID_ID)
+              return _.diag(SPV_ERROR_INVALID_ID, use_block->label())
```

Comment:
This one may have conflicts with the work @alan-baker is doing to split up validate_id. I just wanted to get it up now so that the patch chain was complete.